lib/utils/CSVDatabase.C

```diff
@@ -290,10 +308,15 @@ int
 CSVDatabase::getLineCount(
     const std::string& file_name)
 {
+    int count = 0;
```

Comment:
`CAROM_VERIFY` (although this was not changed in this PR, it seems like a good idea to make it `VERIFY`).